openvino_tensorflow/kernels/encapsulate_op.cc

```diff
@@ -718,7 +719,7 @@ Status NGraphEncapsulateOp::GetExecutable(
 
 Status NGraphEncapsulateOp::Fallback(OpKernelContext* ctx) {
   OVTF_VLOG(1) << "Cluster " << name() << " fallback to native TF runtime ";
-  int64_t start_ns;
+  int64_t start_ns = 0, duration_in_ms;
```

Comment:
Shouldn't we be initializing duration_in_ms to 0 as well?

Reply:
Yes, added.